ipatests/test_webui/test_cert.py

```diff
@@ -226,8 +251,16 @@ def test_search_maximum_serial(self):
         """
         self.init_app()
         self.navigate_to_entity(ENTITY)
+
+        # Find the second lowest serial number
+        result = self.execute_api_from_ui('cert_find', [], {})
+        certs = result['result']['result']
+        serials = [int(cert["serial_number_hex"], 0) for cert in certs]
```

Comment:
There should be a "serial_number" value included in the output which is decimal. It's in the API output anyway.

Reply:
I tried to use that in my first attempt but the call sometimes returns the value with a scientific notation (like 3.12345678e+37) which is not precise enough to allow extraction of the real value.

Reply:
Ok, that is probably worth a bug on its own. Hex is fine.